src/cart/crt_iv.c

```diff
@@ -2536,7 +2567,7 @@ handle_ivupdate_response(const struct crt_cb_info *cb_info)
 	} else {
 		d_sg_list_t *tmp_iv_value;
 
-		if (iv_info->uci_bulk_hdl == CRT_BULK_NULL)
+		if (iv_info->uci_iv_value.sg_nr == 0)
```

Comment:
hmm should this be  uci_bulk_hdl == CRT_BULK_NULL && uci_iv_value.sg_nr == 0?
this case we want when there is no iv value, neither inline nor in bulk